src/app/components/challenge/challengesubmissions/challengesubmissions.component.ts

```diff
@@ -9,13 +9,22 @@ import { Router, ActivatedRoute } from '@angular/router';
 import { SelectphaseComponent } from '../../utility/selectphase/selectphase.component';
 import { InputComponent } from '../../utility/input/input.component';
 
+import {Observable} from 'rxjs';
```

Comment:
Here as well. Recommend you to change your editor's settings.

Reply:
Done